package.json

```diff
@@ -77,6 +78,7 @@
     "@types/node": "^18.7.18",
     "@typescript-eslint/eslint-plugin": "^5.37.0",
     "@typescript-eslint/parser": "^5.37.0",
+    "axios": "^0.27.2",
```

Comment:
this one should be a dependency and not a dev-dependency. Anyone importing the sdk will need this.

Reply:
Moved to dependency